tmt/cli.py

```diff
@@ -673,11 +674,19 @@ def tests_import(
             root = Path(fmf.Tree(str(path)).root)
             common['path'] = str(Path('/') / path.relative_to(root))
         # Store common metadata
-        common_path = path / 'main.fmf'
+        file_name: str = common.get('filename', '') or 'main.fmf'
+        common_path = path / file_name
         tmt.convert.write(common_path, common)
         # Store individual data (as virtual tests)
         for testcase in individual:
-            testcase_path = path / f'{testcase["extra-nitrate"]}.fmf'
+            if nitrate and testcase.get('extra-nitrate'):
+                testcase_path = path / f'{testcase["extra-nitrate"]}.fmf'
+            else:
+                file_name = testcase.get('filename', '')
```

Comment:
Is `''` default needed? `get('filename')` will return `None` as a default value, it's a well-known Python pattern, and it would pass the `if not file_name` test below.

Reply:
I guess I wanted to stick to the type restraints, but sure, None would be fine as well